packages/react/src/components/form/states.tsx

```diff
@@ -0,0 +1,11 @@
+import clsx from "clsx";
+import * as styles from "./form.css";
+import { AuthenticatingProps, Authenticating as AuthenticatingState } from "./authenticating";
+
+export const Authenticating = (props: AuthenticatingProps) => {
```

Comment:
It's quite likely that over time we'll forget why this is here, maybe just a comment explaining the use case of this module?